frontend/src/lib/utils.tsx

```diff
@@ -718,3 +718,27 @@ export function autocorrectInterval(filters: Partial<FilterType>): string {
         return filters.interval
     }
 }
+
+function suffixFormatted(value: number, base: number, suffix: string, maxDecimals: number): string {
+    /* Helper function for compactNumber */
+    const multiplier = 10 ** maxDecimals
+    return `${Math.floor((value * multiplier) / base) / multiplier}${suffix}`
```

Comment:
Wouldn't it be better to round normally?

Reply:
Agreed, I originally used floor because of where we used this in emails, but it makes sense to round normally now. Updated.